src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -320,6 +320,8 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       case Failure(t) =>
         self ! AsyncFailure(t)
     }
+
+    futureCaches
```

Comment:
These caches represent sensitive internal WorkflowManager data that the `sender()` shouldn't have access to.  It might be preferable to have the return type be `Future[Unit]` and return `futureCaches map { _ => () }` (or some other nicer way of throwing away the `Future` success value).